### PR TITLE
feat: drm

### PR DESCRIPTION
**Pull request by dan-ziv**
### Description of the Changes

Drm support for dash adapter (implemented by shaka player).

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/dash-adapter.js [hunk 1]
```diff
@@ -75,6 +76,18 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
...
 
   /**
+   * Checks if dash adapter can play a given drm data.
+   * @param {Array<Object>} drmData - The drm data to check.
+   * @returns {boolean} - Whether the dash adapter can play a specific drm data.
+   * @static
+   */
+  static canPlayDrm(drmData: Array<Object>): boolean {
+    let canPlayDrm = DrmHelper.canPlayDrm(drmData);
+    DashAdapter._logger.debug('canPlayDrm result is ' + canPlayDrm.toString());
```
Comment (OrenMe) [inline src/dash-adapter.js]: Comment should be more descriptive IMHO, canPlayDrm method name is not so informative as just printing DRM (is/is not) support

File: src/drm-helper.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  static setDrmConfig(config: Object, drmData: Object): void {
+    config.drm = {servers: {}};
+    drmData.forEach((drmEntry) => {
+      if (drmEntry.scheme === DrmHelper.DrmSchemes.WIDEVINE) {
+        config.drm.servers[DrmHelper.DrmSchemes.WIDEVINE] = drmEntry.licenseUrl;
+        if (BROWSER === 'Chrome') {
+          config.drm.advanced = {
+            [DrmHelper.DrmSchemes.WIDEVINE]: {
```
Comment (OrenMe) [inline src/drm-helper.js]: @dan-ziv is it suppose to be like this, with an array expression?

Comment (dan-ziv) [inline src/drm-helper.js]: it's not an array expression, it's for computed property key

File: src/dash-adapter.js (hunk 1 above)
Comment (dan-ziv) [inline src/dash-adapter.js]: What do u expect to log in addition for the result?

**Comment by OrenMe** [comment]
@dan-ziv test are failing here as well

**Pull request merged**